zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/entities/AuthorizationEntity.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.entities;
+
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import java.util.List;
+import java.util.Map;
+
+public class AuthorizationEntity implements ExporterEntity<AuthorizationEntity> {
+  private String id;
```

Comment:
❓What will id be here? Where does it come from?

Reply:
Looking at the rest of the code I guess it's the record key

Reply:
Thats correct, at the moment its record key